ctapipe_io_nectarcam/__init__.py

```diff
@@ -81,7 +81,7 @@ def __init__(self, **kwargs):
         self.data = None
         self.log.info("Read {} input files".format(self.multi_file.num_inputs()))
 
-
+    @property
     def subarray(self):
         return self.prepare_subarray_info()
```

Comment:
Processing will very often access this, you probably should do something like

```
self._subarray = self.prepare_subarray_info()
```
 in the `__init__` method and here just `return self._subarray`.